roles/usb_lib/templates/mount.d/70-usb-library.j2

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# Create symlink in DocumentRoot/content to automounted usb drive
+#
+# based on a similar script in the xs-rsync package
+# by Martin Langhoff <martin@laptop.org>
+#
+# and the adaptation for xs-activity-server by Douglas Bagnall
+# <douglas@paradise.net.nz>
+#
+# by Tim Moody tim@timmoody.com
+
+VERBOSE=yes
```

Comment:
@avni @jvonau @tim-moody does this `VERBOSE=yes` line really do anything here?

I suspect not.  It would need to be exported, no?  And the real story is probably in /etc/usbmount/usbmount.conf (i.e. https://github.com/rbrito/usbmount/blob/9a92e7d622662380f4329e0db17e263509715722/usbmount.conf#L53) right?

Reply:
Refer to my PR and see if that is exported or not

Reply:
I definitely agree that `VERBOSE=yes` belongs in `/etc/usbmount/usbmount.conf` as per:

- PR #3916

And I'd recommend @avni remove the `VERBOSE=yes` line here in 70-usb-library.j2, where it (likely?) never did anything.